lua/sql.lua

```diff
@@ -323,7 +323,7 @@ end
 --- Insert to lua table into sqlite database table.
 ---@params tbl string: the table name
 ---@params rows table: rows to insert to the table.
----@return boolean: true incase the table was inserted successfully.
+---@return boolean, integer: true incase the table was inserted successfully, and the last inserted row id
```

Comment:
emmy lua specification says: `boolean|integer` https://emmylua.github.io/annotations/return.html

Currently its not yet supported by [tree-sitter-lua](https://github.com/tjdevries/tree-sitter-lua) and docgen but i will add it :)
oh and we have the same syntax for `@param` https://github.com/tjdevries/tree-sitter-lua/blob/f3cf820399800d821079c89f081f7a0459e8d72b/corpus/documentation.txt#L173-L175 (ignore the space after `|`)

Reply:
Awesome! I've fixed it without a space as emmylua link provided 👍🏽